go/libraries/doltcore/env/actions/table.go

```diff
@@ -21,74 +21,127 @@ import (
 	"github.com/liquidata-inc/dolt/go/libraries/doltcore/env"
 )
 
-func CheckoutAllTables(ctx context.Context, dEnv *env.DoltEnv) error {
+func CheckoutAllTables(ctx context.Context, dEnv *env.DoltEnv) (unknown string, err error) {
```

Comment:
Same comment about the return type

Reply:
And on other variations of this function

Reply:
Removed it from both CheckoutAllTables and CheckoutTableAndDocss: https://github.com/liquidata-inc/dolt/pull/299/files#diff-7bbaaa4ef160f9fffed44cef87371fa5R24